<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Notepad</class>
  <widget class="QMainWindow" name="Notepad">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>640</width>
        <height>480</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Notepad</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QTextEdit" name="textEdit"/>
        </item>
      </layout>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>640</width>
          <height>25</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="actionNew"/>
        <addaction name="actionOpen"/>
        <addaction name="actionSave"/>
        <addaction name="actionSave_as"/>
        <addaction name="actionPrint"/>
        <addaction name="separator"/>
        <addaction name="actionExit"/>
      </widget>
      <widget class="QMenu" name="menuEdit">
        <property name="title">
          <string>Edit</string>
        </property>
        <addaction name="actionCopy"/>
        <addaction name="actionCut"/>
        <addaction name="actionPaste"/>
        <addaction name="separator"/>
        <addaction name="actionUndo"/>
        <addaction name="actionRedo"/>
        <addaction name="separator"/>
        <addaction name="actionZoomOut"/>
        <addaction name="actionZoomIn"/>
      </widget>
      <widget class="QMenu" name="menuHelp">
        <property name="title">
          <string>Help</string>
        </property>
        <addaction name="actionAbout"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuEdit"/>
      <addaction name="menuHelp"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="actionNew"/>
      <addaction name="actionOpen"/>
      <addaction name="actionSave"/>
      <addaction name="actionSave_as"/>
      <addaction name="actionPrint"/>
      <addaction name="separator"/>
      <addaction name="actionCopy"/>
      <addaction name="actionCut"/>
      <addaction name="actionPaste"/>
      <addaction name="actionUndo"/>
      <addaction name="actionRedo"/>
      <addaction name="separator"/>
      <addaction name="actionZoomOut"/>
      <addaction name="actionZoomIn"/>
      <addaction name="separator"/>
      <addaction name="actionAbout"/>
      <addaction name="separator"/>
      <addaction name="actionExit"/>
    </widget>
    <widget class="QStatusBar" name="statusBar"/>
    <action name="actionNew">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/new.png</normaloff>:/images/new.png
        </iconset>
      </property>
      <property name="text">
        <string>New</string>
      </property>
      <property name="toolTip">
        <string>New text file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+N</string>
      </property>
    </action>
    <action name="actionOpen">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/open.png</normaloff>:/images/open.png
        </iconset>
      </property>
      <property name="text">
        <string>Open</string>
      </property>
      <property name="toolTip">
        <string>Open file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+O</string>
      </property>
    </action>
    <action name="actionSave">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png
        </iconset>
      </property>
      <property name="text">
        <string>Save</string>
      </property>
      <property name="toolTip">
        <string>Save file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+S</string>
      </property>
    </action>
    <action name="actionSave_as">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png
        </iconset>
      </property>
      <property name="text">
        <string>Save as</string>
      </property>
      <property name="toolTip">
        <string>Save file as</string>
      </property>
      <property name="shortcut">
        <string>Alt+S</string>
      </property>
    </action>
    <action name="actionPrint">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/print.png</normaloff>:/images/print.png
        </iconset>
      </property>
      <property name="text">
        <string>Print</string>
      </property>
      <property name="toolTip">
        <string>Print file</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+P</string>
      </property>
    </action>
    <action name="actionExit">
      <property name="icon">
        <iconset theme="exit.png" resource="notepad.qrc">
          <normaloff>:/images/exit.png</normaloff>:/images/exit.png
        </iconset>
      </property>
      <property name="text">
        <string>Exit</string>
      </property>
      <property name="toolTip">
        <string>Exit Notepad</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Q</string>
      </property>
    </action>
    <action name="actionCopy">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/copy.png</normaloff>:/images/copy.png
        </iconset>
      </property>
      <property name="text">
        <string>Copy</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+C</string>
      </property>
    </action>
    <action name="actionCut">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/cut.png</normaloff>:/images/cut.png
        </iconset>
      </property>
      <property name="text">
        <string>Cut</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+X</string>
      </property>
    </action>
    <action name="actionPaste">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/paste.png</normaloff>:/images/paste.png
        </iconset>
      </property>
      <property name="text">
        <string>Paste</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+V</string>
      </property>
    </action>
    <action name="actionUndo">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/edit_undo.png</normaloff>:/images/edit_undo.png
        </iconset>
      </property>
      <property name="text">
        <string>Undo</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Z</string>
      </property>
    </action>
    <action name="actionRedo">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/edit_redo.png</normaloff>:/images/edit_redo.png
        </iconset>
      </property>
      <property name="text">
        <string>Redo</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Y</string>
      </property>
    </action>
    <action name="actionZoomOut">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/zoom_out.png</normaloff>:/images/zoom_out.png
        </iconset>
      </property>
      <property name="text">
        <string>Zoom Out</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+-</string>
      </property>
    </action>
    <action name="actionZoomIn">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/zoom_in.png</normaloff>:/images/zoom_in.png
        </iconset>
      </property>
      <property name="text">
        <string>Zoom In</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+=</string>
      </property>
    </action>
    <action name="actionAbout">
      <property name="icon">
        <iconset resource="notepad.qrc">
          <normaloff>:/images/info.png</normaloff>:/images/info.png
        </iconset>
      </property>
      <property name="text">
        <string>About</string>
      </property>
      <property name="toolTip">
        <string>About Notepad</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="notepad.qrc"/>
  </resources>
  <connections/>
</ui>
